packages/js/navigation/src/index.js

```diff
@@ -234,7 +234,10 @@ export function updateQueryString(
  */
 export const addHistoryListener = ( listener ) => {
 	// Monkey patch pushState to allow trigger the pushstate event listener.
-	if ( window.wcNavigation && ! window.wcNavigation.historyPatched ) {
+
+	window.wcNavigation = window.wcNavigation ? window.wcNavigation : {};
```

Comment:
Totally nitpick. I think that if the idea here is to verify whether `window.wcNavigation` is null/undefined, this line could be simplified by using the nullish coalescing operator (`??`).
It would look like this:
```
window.wcNavigation = window.wcNavigation ?? {};
```

Reply:
Good thought @octaedro , done!